index.html

```diff
@@ -6,27 +6,32 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" type="text/css" href="style.css" />
     <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;500;700&display=swap">
-    <title>Portfolio page </title>
+    <title>My Portfolio page </title>
 </head>
 <body>
     <div class="main-one">
         <header>
             <h1 class="mylogo">Ndonyi</h1>
-            <menu>
-                <ul class="home">
+            
+                <ul id="home">
                     <li class="content"><a href="#">Portoflio</a></li>
                     <li class="content"><a href="https://citec-47.github.io/frontendmasters/">About</a></li>
                     <li class="content"><a href="https://citec-47.github.io/Desk-version/">Contact</a></li>
                 </ul>
-            </menu>
-          
+              
+                <ul id="home1">
+                    <h1 id="close">X</h1>
+                    <li class="content1"><a href="#">Portoflio</a></li>
+                    <li class="content1"><a href="https://citec-47.github.io/frontendmasters/">About</a></li>
+                    <li class="content1"><a href="https://citec-47.github.io/Desk-version/">Contact</a></li>
+                </ul>
```

Comment:
- [ ] your creativity is really impressive :hugs:
- However, for the sake of this project, we have to strive to match the Figma wireframe. To achieve this, we have a few changes to implement:
  - [ ] Let's make all your `a` to redirect users to corresponding sections on your page, and not some external links.
  - [ ] Your `hamburger menu` is enabled upon the loading of the page automatically. You can check this [video](https://www.awesomescreenshot.com/video/15396554?key=4872f066efb9f16f15873f200b28f5bb).

Reply:
Good job overall 💯  but as requested by the previous reviewer the `a` tags should redirect to corresponding sections on your page, and not some external links. 🔗